code/modules/projectiles/projectile/beams.dm

```diff
@@ -68,6 +68,7 @@ var/list/beam_master = list()
 	flag = "laser"
 	eyeblur = 4
 	fire_sound = 'sound/weapons/Laser.ogg'
+	anchored = TRUE
```

Comment:
Aren't projectiles normally anchored?

Reply:
ah fuck

should explosions throw anchored obj/ should throw_at still throw it even when its anchored?

Reply:
Explosions definitely shouldn't throw anchored objects
`throw_at()` is up for debate